.gitignore

```diff
@@ -18,5 +18,4 @@ rustup/*
 compile_commands.json
 
 # cmps (https://github.com/leonmatthes/cmps)
-.cmps/
-
+.cmps/
```

Comment:
likely a new line missing at the end of the file here which is causing a diff

Reply:
```suggestion
.cmps/

```